javascript/builtins/Intl.json

```diff
@@ -1082,6 +1089,70 @@
               }
             }
           },
+          "formatToParts": {
+            "__compat": {
+              "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat/formatToParts",
+              "support": {
+                "webview_android": {
+                  "version_added": false
+                },
+                "chrome": {
+                  "version_added": "63",
+                  "flag": {
+                    "type": "preference",
+                    "name": "harmony-number-format-to-parts"
+                  },
+                  "notes": "Need to set the flag on the commandline via --js-flag"
+                },
+                "chrome_android": {
+                  "version_added": "63",
+                  "flag": {
+                    "type": "preference",
+                    "name": "harmony-number-format-to-parts"
+                  },
+                  "notes": "Need to set the flag on the commandline via --js-flag"
+                },
+                "edge": {
+                  "version_added": null
+                },
+                "edge_mobile": {
+                  "version_added": null
+                },
+                "firefox": {
+                  "version_added": null
```

Comment:
It's in Firefox 58, see https://bugzilla.mozilla.org/show_bug.cgi?id=1403319

Reply:
Good to know. I'll change that.